homeassistant/components/roku/media_player.py

```diff
@@ -69,6 +72,41 @@ async def async_setup_entry(hass, entry, async_add_entities):
     )
 
 
+def browse_media_library(channels: bool = False) -> dict:
+    """Create response payload to describe contents of a specific library."""
+    library_info = {
+        "title": "Media Library",
+        "media_content_id": "library",
+        "media_content_type": "library",
+        "can_play": False,
+        "can_expand": True,
+        "children": [],
+    }
+
+    library_info["children"].append(
+        {
+            "title": "Apps",
+            "media_content_id": "apps",
+            "media_content_type": "apps",
+            "can_expand": True,
+            "can_play": False,
+        }
+    )
+
+    if channels:
+        library_info["children"].append(
+            {
+                "title": "Channels",
+                "media_content_id": "channels",
+                "media_content_type": "channels",
```

Comment:
Each media content type needs to be used from `media_player/const.py`

Reply:
are referring to the key here as i dont think channels is valid type since custom here for browser

Reply:
Then let's define one there